### ISSUE TITLE
[Bug]: nuxtr.vueFiles.firstTag "Unknown Configuration Setting", however the setting does actually work

### ISSUE THREAD

**Issue by pahenn** [nuxtrdev/nuxtr-vscode#5]
### Environment

```
Nuxi 3.4.3                                                                                              11:56:41 AM
                                                                                                        11:56:42 AM
RootDir: /Users/patrick/personal/fresh-mn3                                                              11:56:42 AM
Nuxt project info: (copied to clipboard)                                                                11:56:42 AM

------------------------------
- Operating System: Darwin
- Node Version:     v18.14.2
- Nuxt Version:     3.4.3
- Nitro Version:    2.4.0
- Package Manager:  pnpm@8.5.0
- Builder:          vite
- User Config:      modules, css, postcss, devtools
- Runtime Modules:  @vueuse/nuxt@10.1.2
- Build Modules:    -
------------------------------

👉 Report an issue: https://github.com/nuxt/nuxt/issues/new                                             11:56:42 AM

👉 Suggest an improvement: https://github.com/nuxt/nuxt/discussions/new

👉 Read documentation: https://nuxt.com
```

### What went wrong?

This setting actually works, but VSCode thinks it does not exist.

<img width="315" alt="image" src="https://github.com/nuxtrdev/nuxtr-vscode/assets/7787945/e459bab2-484d-4275-abb6-541c27672bdb">

When adding this setting to my settings.json, the setting is reported as not found.

### How to reproduce it?

- Install Nuxtr from VSCode Extension Marketplace
- Try to add the setting to settings.json

**Comment by adhamfarrag** [comment]
Hello @pahenn!
Fixed and will be available with next release.
We wanted to group settings as we scale the project to make onboarding/learning curve easier. Ended up not including this setting by mistake while still relying on it for our logic. This is why VSCode is giving this error. :) 

Thanks for reporting it! Will revisit the issue once it's released :)

**Comment by adhamfarrag** [comment]
Hello @pahenn!

This issue should be fixed with V 0.1.1. Please, update Nuxtr!
Thanks for reporting this issue :)
